repo/content/content_manager.go

```diff
@@ -852,9 +887,13 @@ func (bm *WriteManager) getOverlayContentInfoReadLocked(contentID ID) (*pendingP
 }
 
 // +checklocksread:bm.mu
-func (bm *WriteManager) getContentInfoReadLocked(ctx context.Context, contentID ID) (*pendingPackInfo, Info, error) {
+// NOTE: previousWriteTime is written even if error is ErrContentNotFound
+// in cases where the content was previously forgotten.
+func (bm *WriteManager) getContentInfoReadLocked(ctx context.Context, contentID ID, previousWriteTime *int64) (*pendingPackInfo, Info, error) {
```

Comment:
Given that this function is already returning 3 values, why not return a fourth, instead of mixing _output_ parameters and return values?

It seems that it would also make it simpler to ignore where it needs to be ignored. Thoughts?

Reply:
done